static/app/components/events/autofix/autofixSteps.tsx

```diff
@@ -111,13 +111,33 @@ function Progress({
   progress: AutofixProgressItem | AutofixStep;
   runId: string;
 }) {
+  function replaceHeadersWithBold(markdown) {
```

Comment:
minor nit: let's type this
```suggestion
  function replaceHeadersWithBold(markdown: string) {
```